src/silx/gui/widgets/FrameBrowser.py

```diff
@@ -312,3 +312,100 @@ def setValue(self, value):
     def value(self):
         """Get selected value"""
         return self._slider.value()
+
+class FrameRateWidgetAction(qt.QWidgetAction):
+    def __init__(self, parent):
+        super().__init__(parent)
+        self._build()
+        
+    def _build(self):
+        widget = qt.QWidget()
+        layout = qt.QHBoxLayout()
+        widget.setLayout(layout)
+        self.line_edit = qt.QLineEdit()
+        layout.addWidget(qt.QLabel("FPS:"))
+        layout.addWidget(self.line_edit)
+        self.setDefaultWidget(widget)
+        
+class IntervalsWidgetAction(qt.QWidgetAction):
+    def __init__(self, parent):
+        super().__init__(parent)
+        self._build()
+        
+    def _build(self):
+        widget = qt.QWidget()
+        layout = qt.QHBoxLayout()
+        widget.setLayout(layout)
+        self.spinbox = qt.QSpinBox()
+        layout.addWidget(qt.QLabel("Interval:"))
+        layout.addWidget(self.spinbox)
+        self.setDefaultWidget(widget)
+
+class PlayButtonContextMenu(qt.QMenu):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._build()
+
+    def _build(self):
+        self.line_edit_action = FrameRateWidgetAction(self)
+        self.interval_action = IntervalsWidgetAction(self)
+        self.addAction(self.line_edit_action)
+        self.setFrameRate(value="10")
+        self.addAction(self.interval_action)
+        self.setInterval(value=1)
+        
+    def setFrameRate(self, value:str):
+        self.line_edit_action.line_edit.setText(value)
+        
+    def getFrameRate(self):
+        return int(self.line_edit_action.line_edit.text())
+    
+    def setInterval(self, value:int):
+        self.interval_action.spinbox.setValue(value)
+    
+    def getInterval(self):
+        return int(self.interval_action.spinbox.value())
+        
+class HorizontalSliderWithBrowserPlay(HorizontalSliderWithBrowser):
```

Comment:
I would add the play/pause feature in the `HorizontalSliderWithBrowser`.
The new feature is quite consistent with what's already there.

We can consider adding methods to show/hide the Play button if the play feature needs to be disabled at some point.